core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestService.java

```diff
@@ -402,6 +402,15 @@ protected String stringifyPepperAddress(PepperKitAddress kitAddress) {
      */
     private void saveKitStatus(KitRequest kitRequest, PepperKit pepperKit, Instant pepperStatusFetchedAt) {
         KitRequestStatus priorStatus = kitRequest.getStatus();
+
+        if(priorStatus.equals(KitRequestStatus.DEACTIVATED)) {
+            // if the kit has been deactivated in Juniper, we no longer need to update the status
+            // based on what DSM thinks. this allows us to independently deactivate kits in Juniper
+            // without having to worry about DSM status updates.
+            log.info("Skipping status update for deactivated kit request %s".formatted(kitRequest.getId()));
```

Comment:
can we change this to not log anything (or maybe debug) if the DSM status matches, but to log a warning if the statuses do not match?